tests/v2/validation/snapshot/snapshot.go

```diff
@@ -231,54 +219,34 @@ func restoreRKE1(t *testing.T, client *rancher.Client, rke1Snapshot initialSnaps
 
 		podErrors := pods.StatusPods(client, clusterID)
 		assert.Empty(t, podErrors)
-		require.Equal(t, rke1Snapshot.kubernetesVersion, clusterResp.RancherKubernetesEngineConfig.Version)
-
-		if etcdRestore.SnapshotRestore == kubernetesVersion || etcdRestore.SnapshotRestore == all {
-			clusterResp, err = client.Management.Cluster.ByID(clusterID)
-			require.NoError(t, err)
-			require.Equal(t, rke1Snapshot.kubernetesVersion, clusterResp.RancherKubernetesEngineConfig.Version)
 
-			if etcdRestore.ControlPlaneUnavailableValue != "" && etcdRestore.WorkerUnavailableValue != "" {
-				logrus.Infof("Control plane unavailable value is restored to: %s", clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane)
-				logrus.Infof("Worker unavailable value is restored to: %s", clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableWorker)
-
-				require.Equal(t, rke1Snapshot.initialControlPlaneUnavailable, clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane)
-				require.Equal(t, rke1Snapshot.initialWorkerUnavailable, clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableWorker)
-			}
-		}
+		require.Equal(t, cluster.RancherKubernetesEngineConfig.Version, clusterResp.RancherKubernetesEngineConfig.Version)
+		require.Equal(t, cluster.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane, clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane)
+		require.Equal(t, cluster.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableWorker, clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableWorker)
```

Comment:
this logic is now checking that the values on the cluster before the restore == values on the cluster after the restore. But since we depend on the test to set controlplane/workerUnavailable, this shouldn't be true if the test is configured to set them. I think this should revert to the previous logic.

Reply:
I'm not following. This check is going to make sure what was set before the restore is the same after the restore happened.